packages/schema/src/schema.graphql

```diff
@@ -283,6 +284,16 @@ type BlockQuote @type(id: "custom/quote") {
   image: MediaImage @resolveEditorBlockMedia
 }
 
+type BlockInfoGrid @type(id: "custom/info-grid") {
+  items: [BlockInfoGridItem]! @resolveEditorBlockChildren
+}
+
+type BlockInfoGridItem @default @value {
+  icon: String! @resolveEditorBlockAttribute(key: "icon")
+  textContent: BlockMarkup @resolveEditorBlockChildren @seek(pos: 0)
+  cta: BlockCta @resolveEditorBlockChildren @seek(pos: 1)
```

Comment:
🐛 From the grid item block source:
```tsx
          <InnerBlocks
            templateLock={false}
            allowedBlocks={['custom/heading', 'core/paragraph', 'custom/cta']}
```
This means that the block can have unlimited amount of children in any order. Tough, they are limited to 3 types.

Yet the schema here expects a fixed set of 3 blocks (2 first are grouped into 1).

I guess `templateLock` should be `true`?